common/src/lib.rs

```diff
@@ -213,11 +219,23 @@ impl Program {
     }
 
     pub fn is_exited(&self) -> bool {
-        matches!(self, Program::Exited(_))
+        matches!(
+            self,
+            Program::Exited {
+                inheritor: _,
+                block_number: _
+            }
```

Comment:
Why match `inheritor` and `block_number`?
```suggestion
            Program::Exited {..}
```

Reply:
agree with @shamilsan

Reply:
done